content/stable-rc/index.html

```diff
@@ -10,8 +10,6 @@
 <p>
 LTS Release candidate is the build intended for the upcoming LTS release of Jenkins. <a href="https://wiki.jenkins-ci.org/display/JENKINS/Release+Process">See Wiki for more about what it means.</a>
 </p>
-<p>Upcoming version: <b><?php include "/srv/releases/jenkins/version-stable-rc.txt"?></b></p>
```

Comment:
@olivergondza if you can point me to where I can fetch the RC stable version from, I can incorporate it. [scripts/fetch-external-resources](https://github.com/jenkins-infra/jenkins.io/blob/master/scripts/fetch-external-resources) grabs some files to provide some variables at site generation time (e.g. `site.jenkins.latest`)